saladItem.js

```diff
@@ -1,3 +1,68 @@
 import React, { useReducer, useContext } from 'react';
 import PropTypes from 'prop-types';
 import { createUseStyles } from 'react-jss';
+
+import UserContext from '../User/User';
+import { SaladContext } from '../SaladMaker/SaladMaker';
+
```

Comment:
General Comments:
- Would like to add a lot of semicolons. While not strictly necessary, I was taught it is best practice, and it improves readability.
- This component uses a very simple counter function. It is probably a good idea to have a way to either reset and/or decrement the counter as well in case the user decides they don't want that particular salad item or they mis-clicked it. In this case it seems these things were excluded for simplicity. However, buttons for these features could be added as a conditional render that only shows up once the user and initially added a salad item.